azerothcore/azerothcore-wotlk
# fix(DB/creature): Beaten Corpse faction and flags

Beaten Corpse (id: 10668) was assigned the wrong faction and flags
which made its nameplate appear grey instead of green for Horde players

According to https://classic.wowhead.com/npc=10668/beaten-corpse, the
creature should appear friendly to Horde players and hostile towards
Alliance players.



## Changes Proposed:
- Modify faction and dynamic flags of creature id 10668

## Issues Addressed:

- Closes #6318
- Closes https://github.com/chromiecraft/chromiecraft/issues/841

## SOURCE:

https://classic.wowhead.com/npc=10668/beaten-corpse
https://youtu.be/g-1A-JD2UMQ?t=23971

## Tests Performed:

- Tested in-game. Dialogue options still work and the "Lost in Battle" quest is still completable. Creature can't be damaged.

## How to Test the Changes:


1. `go c 19221`
2. confirm that creature appears friendly towards Horde players and hostile towards Alliance players
3. confirm that the creature can't be damaged


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
